### PR TITLE
README upgrade

### PR DESCRIPTION
**Pull request by mieubrisse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,55 +26,31 @@ In your Docker engine you'll now see:
...
-    1. The initializer you'll need to rerun `scripts/rebuild_initializer_binary.sh`
-    2. The controller you'll need to rerun `scripts/rebuild_controller_image.sh`
-    3. The commons code you'll need to rerun both
-
-### Helpful Tip
-You can use the following alias to stop & remove Docker containers of a certain type:
+Developing Locally
+------------------
+### Architecture
+This repo uses the [Kurtosis architecture](https://github.com/kurtosis-tech/kurtosis), so you should first go through the tutorial there to familiarize yourself with the core Kurtosis concepts.
```
Comment (galenmarchetti) [inline README.md:32]: how are we going to handle this if we make the repo private / closed source?

File: README.md [hunk 2]
```diff
@@ -15,55 +26,31 @@ In your Docker engine you'll now see:
...
 
-```
-# alias for clearing kurtosis containers 
-clear_containers() {  docker rm $(docker stop $(docker ps -a -q --filter ancestor="$1" --format="{{.ID}}")); } 
-alias cclear=clear_containers
-```
+In this implementation of Kurtosis, we have:
+* `AvaService` and `GeckoService` as our service interfaces
```
Comment (galenmarchetti) [inline README.md:35]: i think its worth explaining a bit what the difference is (one is generalized ava, one is the gecko client implementation)

File: README.md [hunk 3]
```diff
@@ -15,55 +26,31 @@ In your Docker engine you'll now see:
...
-cclear ${GECKO_IMAGE} 
-cclear ${CONTROLLER_IMAGE} 
-```
-
-## Writing Tests
-This repo uses the [Kurtosis architecture](https://github.com/kurtosis-tech/kurtosis), so you'll want to be familiar with the concepts there. In this implementation:
-* The `AvaTestSuite` struct defines the tests that will be run
-* The `AvaTestSuite` struct gets registered with the initializer and the controller in `initializer/main.go` and `controller/main.go` respectively
-* The tests, the networks the tests will run against, and the services the networks are composed of live in the `commons` package
+Additionally, this repo also contains a Gecko client (which should probably be moved to the Gecko repo).
```
Comment (galenmarchetti) [inline README.md:44]: worth describing this as an golang RPC to differentiate from "Gecko client" as in gecko itself

File: README.md (hunk 1 above)
Comment (mieubrisse) [reply README.md:32]: I think that's part of the "how do we give clients of Kurtosis the documentation". The Ava implementation is a client, so they'd have access to the documentation (however it's distributed)

**Pull request merged**